src/planwise/component/projects.clj

```diff
@@ -89,3 +89,18 @@
             stats (compute-project-stats service project)]
         (update-project* tx {:project-id project-id
                              :stats (pr-str stats)})))))
+
+(defn update-project
+  [service project]
+  (jdbc/with-db-transaction [tx (get-db service)]
+    (let [project-id (:id project)
+          goal       (:goal project)
+          filters    (:filters project)
+          stats      (:stats project)
+          params     {:project-id project-id
+                      :goal       goal
+                      :filters    (some-> filters pr-str)
+                      :stats      (some-> stats pr-str)}
+          result     (update-project* tx params)]
+      (when (= 1 result)
+        (load-project tx project-id)))))
```

Comment:
I think I found it: https://github.com/instedd/planwise/pull/85/commits/87cc9523c49206b181a369d188ef8ff2b73bc1a4#diff-46159c4ae07f05e7b239fba08a88720cR44